src/rootPages/Designer/ui_work_object_workspace_popupNewDataField.js

```diff
@@ -72,108 +81,156 @@ export default function (AB) {
+                           view: "button",
+                           id: ids.buttonMaximize,
+                           label: L("<span class=\"webix_icon\"><i class=\"nomargin fa fa-expand\"></i></span>"),
+                           css: "webix_transparent",
+                           width: 40,
+                           click: () => {
+                              this.buttonMaximize();
+                           }
+                       },
+                       {
+                           view: "button",
+                           id: ids.buttonMinimize,
+                           label: L("<span class=\"webix_icon\"><i class=\"nomargin fa fa-compress\"></i></span>"),
+                           hidden: true,
+                           css: "webix_transparent",
+                           width: 40,
+                           click: () => {
+                              this.buttonMinimize();
+                           }
+                       },
+                       {
+                           view: "button",
+                           label: L("<span class=\"webix_icon\"><i class=\"nomargin fa fa-times\"></i></span>"),
+                           css: "webix_transparent",
+                           width: 40,
+                           click: () => {
+                              this.buttonCancel();
+                           },
+                           on: {
+                              onAfterRender() {
+                                 UIClass.CYPRESS_REF(this);
+                              },
+                           }
+                        }
+                     ]
+                  }
                ],
             },
             // ready: function () {
             //  console.error('ready() called!!!')
             //  _logic.resetState();
             // },
-
             body: {
-               view: "scrollview",
-               scroll: "y",
-               css: "ab-add-fields-popup",
-               borderless: true,
-               body: {
-                  type: "form",
-                  rows: [
-                     {
-                        view: "richselect",
-                        id: ids.types,
-                        label: L("Field type"),
-                        labelWidth: uiConfig.labelWidthLarge,
-                        options: [
-                           //We will add these later
-                           { id: "temporary", view: "temporary" },
-                        ],
-                        on: {
-                           onChange: (id /* , ev, node */) => {
-                              this.onChange(id);
-                           },
+               rows: [
+                  {
+                     id: ids.chooseFieldType,
+                     rows: [
+                        {
+                           view: "search",
+                           id: ids.searchBar,
+                           placeholder: "Search by title...",
```

Comment:
Wrap this in the L() function so we can translate it later... 

`placeholder: L("Search by title..."),`